src/header/header.scss

```diff
@@ -190,181 +136,191 @@
+    cursor: not-allowed;
+
     .iui-header-button-icon {
       filter: grayscale(1) opacity(0.7);
     }
   }
 
   @at-root {
-    .iui-page-header.iui-slim {
-      > .iui-left {
-        .iui-button {
-          .iui-header-button-icon.iui-header-button-icon {
-            width: $iui-icons-small;
-            height: $iui-icons-small;
-            padding: $iui-xxs;
-          }
+    .iui-slim {
+      .iui-header-breadcrumbs-button {
+        .iui-header-button-icon {
+          width: $iui-icons-small;
+          height: $iui-icons-small;
+          padding: $iui-xxs;
+          margin: 0 $iui-xs;
+        }
+
+        img.iui-header-button-icon {
+          width: $iui-icons-small + ($iui-xxs * 2);
+          height: $iui-icons-small + ($iui-xxs * 2);
+          padding: 0;
         }
 
-        .iui-button-label {
-          .iui-description {
-            @include iui-display-none;
-          }
+        .iui-header-breadcrumbs-button-description {
+          @include iui-display-none;
         }
       }
     }
   }
 }
 
+@mixin iui-header-button {
+  @include iui-button;
+  @include iui-button-borderless;
+  @include iui-header-button-reset;
+  flex-shrink: 0;
+}
+
+@mixin iui-header-button-icon($size: $iui-icons-large, $border-radius: $iui-border-radius) {
+  width: $size;
+  height: $size;
+  border-radius: $border-radius;
+  @media (prefers-reduced-motion: no-preference) {
+    transition: width $iui-speed-fast ease-out, height $iui-speed-fast ease-out;
+  }
+}
+
 @mixin iui-header-split-button {
+  max-width: 25vw;
+  transition: background-color $iui-speed-fast ease-out;
+
   &,
   > * {
     height: 100%;
     display: flex;
     overflow: hidden;
   }
 
-  max-width: 25vw;
-
-  .iui-button {
-    @include iui-header-button;
+  .iui-header-breadcrumbs-button {
     padding: 0 $iui-xs;
   }
 
   > *:first-child {
-    .iui-button {
+    .iui-header-breadcrumbs-button {
       padding-left: $iui-sm;
+
+      &:hover {
+        background-color: transparent;
+      }
     }
   }
 
+  &.iui-disabled {
+    cursor: not-allowed;
+  }
+
   &.iui-active {
     @include iui-header-button-active;
 
     // Removing button hovers and backgrounds and adding last child hover
-    .iui-button {
-      &.iui-borderless {
-        &,
-        &:hover {
-          background-color: transparent;
-        }
-
-        &::after {
-          display: none;
-        }
+    .iui-header-breadcrumbs-button {
+      &,
+      &:hover {
+        background-color: transparent;
       }
-    }
 
-    > *:last-child {
-      > .iui-button:hover {
-        @include themed {
-          background-color: rgba(t(iui-color-foreground-primary-rgb), t(iui-opacity-6));
-        }
+      &::after {
+        display: none;
       }
     }
 
-    &.iui-disabled {
-      .iui-button {
-        &:last-child:hover {
-          @include themed {
-            background-color: transparent;
-          }
-        }
+    > *:last-child > .iui-header-breadcrumbs-button:hover {
```

Comment:
0-5-0 🤔

Reply:
I was able to reduce specificity of this by applying a class directly.